src/redux/books/Book.js

```diff
@@ -1,53 +1,43 @@
 /* eslint-disable linebreak-style */
-const ADD_BOOK = "bookStore/books/ADD_BOOK";
-const DELETE_BOOK = "bookStore/books/DELETE_BOOK";
-const GET_BOOK = "bookStore/books/GET_BOOK";
-
-export const initState = {
-  books: [
-    {
-      id: 0,
-      genre: "Action",
-      title: "The Hunger Games",
-      author: "Suzanne Collins",
-    },
-    {
-      id: 1,
-      genre: "Science Fiction",
-      title: "Dune",
-      author: "Frank Herbert",
-    },
-    {
-      id: 2,
-      genre: "Economy",
-      title: "Capital in the Twenty-First Century",
-      author: "Suzanne Collins",
-    },
-  ],
-};
 
-export const addBook = (extraBook) => ({
-  type: ADD_BOOK,
-  extraBook,
-});
+const baseId =
+  "https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/";
+const appId = "j0OQb9mCsBBmyNW5xHF4";
```

Comment:
- [ ] Let's fix the address of the API, currently is giving a 404 error when someone tries to add a book to the store. [All the available endpoints are here.](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739)

![image](https://user-images.githubusercontent.com/79114233/142441350-67263d2f-4850-4cc6-a78a-b955cc688b92.png)

Reply:
* Hi this error still persists kindly fix it 🙏